lib/json/serialization.nit

```diff
@@ -461,25 +472,34 @@ class JsonDeserializer
 	#     serialize
 	#
 	#     var error: String
+	#     var related_data: MyData
 	# end
 	#
 	# class MyJsonDeserializer
 	#     super JsonDeserializer
 	#
 	#     redef fun class_name_heuristic(json_object)
```

Comment:
I would pass `attributes_path.last` as argument here for two reasons:

* cleaner clients, remove the `not_empty` && `last` combo
* better understanding on how to use the method itself, the parameter acts as a documentation for new users: "you can use this information too"